cschema.cql

```diff
@@ -8,9 +8,15 @@ CREATE KEYSPACE gallerymodule
 
 USE gallerymodule;
 
-CREATE TABLE listings (
-  id SERIAL PRIMARY KEY,
-  src VARCHAR(100) NOT NULL,
-  description VARCHAR(100) NOT NULL,
+CREATE TABLE images (
+  id            UUID PRIMARY KEY,
```

Comment:
Reasoning for using UUID? I don't know too much about it.

Reply:
From what I've read, I believe it's to make the id unique